### PR TITLE
api call

### PR DESCRIPTION
**Pull request by Adexxy**

### CODE REVIEW

File: kaisen/kaisen/src/taiwo/api_call.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import requests
+
+note1 = "6.5095° N, 3.3711° E"
+note2 = "https://api.open-meteo.com/v1/forecast?latitude={latitude}&longitude={longitude}&hourly=temperature_2m"
+note3 = "https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&current=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m"
+note4 = "https://api.open-meteo.com/v1/forecast?latitude=6.5095&longitude=3.3711&hourly=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m"
```
Comment (tofetpuzo) [inline kaisen/kaisen/src/taiwo/api_call.py:6]: I think we would have to give better variable names for them.

File: kaisen/kaisen/src/taiwo/api_call.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+latitude = 6.5095  # Yaba, Lagos latitude
+longitude = 3.3711  # Yaba, Lagos longitude
+url = f"https://api.open-meteo.com/v1/forecast?latitude={latitude}&longitude={longitude}&hourly=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m"
+
+response = requests.get(url)
+data = response.json()
```
Comment (tofetpuzo) [inline kaisen/kaisen/src/taiwo/api_call.py:15]: I think same goes for this ones too

File: kaisen/kaisen/src/taiwo/api_call.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+if response.status_code == 200:
+    temperature_data = data['hourly']['temperature_2m']
+    humidity_data = data['hourly']['relative_humidity_2m']
+    wind_data = data['hourly']['wind_speed_10m']
+    print(f"Temperature forecast for the next few hours: {temperature_data[:5]}")  # Print first 5 hours
+    print(f"Relative humidity forecast for the next few hours: {humidity_data[:5]}")  # Print first 5 hours
+    print(f"Wind speed forecast for the next few hours: {wind_data[:5]}")  # Print first 5 hours
```
Comment (tofetpuzo) [inline kaisen/kaisen/src/taiwo/api_call.py:23]: I also will introduce the concept of logging , using the [`logging module lib`](https://docs.python.org/3/library/logging.html) from python

**Review by tofetpuzo — COMMENTED** [review]
I have included some comments for discussion

File: kaisen/kaisen/src/taiwo/api_call.py (hunk 1 above)
Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/api_call.py:6]: okay

File: kaisen/kaisen/src/taiwo/api_call.py (hunk 2 above)
Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/api_call.py:15]: alright

File: kaisen/kaisen/src/taiwo/api_call.py (hunk 3 above)
Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/api_call.py:23]: ok.

**Review by tofetpuzo — APPROVED** [review]

**Pull request merged**